controllers/shooter.py

```diff
@@ -37,14 +39,13 @@ def aiming(self, initial_call) -> None:
         if initial_call:
             self.aim()
         else:
-            if (
-                self.chassis.at_desired_heading()
-                and self.shooter_component.is_ready()
-                and self.in_range()
-            ):
-                self.next_state(self.firing)
+            if self.chassis.at_desired_heading():
+                self.status_lights.in_range()
```

Comment:
Move this check to the `note.py` state machine in the `holding_note` state. At the moment the status lights only get updated when the command to fire is given. They should be updating all the time a note is held.